packages/cli-dashboard/package.json

```
@@ -24,7 +24,7 @@
   },
   "homepage": "https://github.com/GoogleChromeLabs/ps-analysis-tool#readme",
   "scripts": {
-    "dev": "webpack --config ../../dashboard.webpack.config.cjs --watch",
+    "dev": "NODE_ENV=development webpack serve --config ../../dashboard.webpack.config.cjs",
```

```suggestion
    "dev": "webpack serve --config ../../dashboard.webpack.config.cjs  --open",
```

The default is `development`, so will `NODE_ENV=development` be required? Also, let's open it after serving.

https://github.com/GoogleChromeLabs/ps-analysis-tool/blob/2b89c2ef1685d1f5700d021b888a936cf8a0c6c4/webpack.shared.cjs#L19